subprojects/common/problem/src/main/kotlin/com/avito/android/Problem.kt

```diff
@@ -12,45 +12,45 @@ internal data class Problem(
     val throwable: Throwable? = null
 ) {
 
-    fun newBuilder(): Builder {
+    public fun newBuilder(): Builder {
         return Builder(this)
     }
 
-    class Builder(private val shortDescription: String, private val context: String) {
+    public class Builder(private val shortDescription: String, private val context: String) {
 
         private var because: String? = null
         private val possibleSolutions = mutableListOf<String>()
         private var documentedAt: String? = null
         private var throwable: Throwable? = null
 
-        constructor(source: Problem) : this(source.shortDescription, source.context) {
+        internal constructor(source: Problem) : this(source.shortDescription, source.context) {
```

Comment:
```suggestion
        private constructor(source: Problem) : this(source.shortDescription, source.context) {
```
?

Reply:
can't ; private constuctor is not available for `newBuilder()`